keyboards/preonic/keymaps/pezhore/keymap.c

```diff
@@ -0,0 +1,328 @@
+/* Copyright 2015-2017 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+
+enum preonic_layers {
+  _QWERTY,
+  _FN,
+  _NUMPAD,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum preonic_keycodes {
+  QWERTY = SAFE_RANGE,
+  FN,
+  NUMPAD,
+  LOWER,
+  RAISE,
+  SHRUG,  // ¯\_(ツ)_/¯
+  TFLIP,  // (╯°□°)╯︵ ┻━┻
+  POOP,   // 💩
+  DPOINT, // (ಠ_ಠ)
+  STRUT   // ᕕ( ᐛ )ᕗ
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
Just to keep the keymap aligned nicely.

```suggestion
#define TG_NUMP TG(_NUMPAD)

const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```